Sources/plank/Cli.swift

```diff
@@ -35,6 +39,8 @@ extension FlagOptions : HelpCommandOutput {
             "    --\(FlagOptions.objectiveCClassPrefix.rawValue) - The prefix to add to all generated class names.",
             "    --\(FlagOptions.outputDirectory.rawValue) - The directory where generated code will be written.",
             "    --\(FlagOptions.printDeps.rawValue) - Just print the path to the dependent schemas necessary to generate the schemas provided and exit.",
+            "    --\(FlagOptions.noRecursive.rawValue) - Don't generate files recursively. Only generate the one file I ask for.",
+            "    --\(FlagOptions.onlyRuntime.rawValue) - Only the plank generate runtime files and exit.",
```

Comment:
nit: ordering of words is a bit off here. Should be "Only generate runtime files and exit."